source/fundamentals/auth.txt

```diff
@@ -211,20 +202,6 @@ mechanism:
       .. literalinclude:: /examples/generated/AuthTest.snippet.scram-sha-1-cred.kt
          :language: kotlin
 
-.. _mongodb-cr-auth-mechanism:
```

Comment:
S: can't comment on the lines, but you can remove the note from line 158-160 about SCRAM-SHA-1 as the default